src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,153 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
```

Comment:
For the future, remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```